dashboard/app/models/levels/aichat.rb

```diff
@@ -37,7 +38,9 @@ def self.create_from_level_builder(params, level_params)
         user: params[:user],
         game: Game.aichat,
         level_num: 'custom',
-        properties: {}
+        properties: {
```

Comment:
Nice! What do you think of adding the default value to `standalone_video.rb`?

Reply:
Done! Added a note as well since I'm not updating all existing levels to have this property at this point.